apps/react/src/utils.js

```diff
@@ -0,0 +1,31 @@
+export function getList() {
+	const l = Math.pow(10, 4);
+	const arr = [];
+	for (let i = 0; i < l; i++) {
+		arr.push({
+			id: Math.random().toString(),
+			content: (Date.now() * Math.random()).toString(),
+		})
+	}
+
+	return arr;
+}
+
+let startTime;
+let type;
+
+export const FN = {
+	loadList(setList) {
+		setList(getList());
+		startTime = performance.now();
+		type = 'load';
+	},
+	clearList(setList) {
+		setList([]);
+		startTime = performance.now();
+		type = 'clear';
+	},
+	measure(setTimers) {
+		setTimers(state => ({...state, [type]: (performance.now() - startTime).toFixed(3) }))
+	}
+}
```

Comment:
this file doesn't make sense, because the "getList" is a pure function that should be similar for both vue and react to get the same content.
the FN object is a react-specific way to try and improve the rendering time.

Reply:
Ha OK I understand. then the measure should simply return `type` and `performance.now()`, and `vue`/`react` will use it internally in the component

Reply:
This is a performant way to measure time in react.
Vue is much simpler and the measure is more precise.